datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -183,6 +184,8 @@ def default_json_event_encoding(o):
         return float(o)
     if isinstance(o, (datetime.date, datetime.datetime)):
         return o.isoformat()
+    if isinstance(o, IPv4Address):
```

Comment:
We've been seeing json serialization errors for a long time now due to some activity payloads containing `IPv4Address` objects